components/vf-sass-config/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 2.0.0
```

Comment:
Doesn't seem like a breaking change, but maybe I'm overlooking something ...

Reply:
I guess it's not as we're not really breaking functionality here -- just adding to it - can happily be pulled back to `1.5.0`

Reply:
taking a look at this - I think because the HTML or Nunjucks has changed dramatically - it warrants the major / breaking version change.